package.json

```
@@ -215,6 +215,7 @@
     "@types/react-loadable": "5.5.2",
     "@types/react-virtualized-auto-sizer": "1.0.0",
     "@types/sockjs-client": "^1.1.1",
+    "@types/uuid": "^8.3.0",
```

Please use specific versions
```suggestion
    "@types/uuid": "8.3.0",
```